### ISSUE TITLE
Request for pretrained model for fine-grained level PartNet segmentation

### ISSUE THREAD

**Issue by guochengqian** [daerduoCarey/partnet_seg_exps#8]
Hello. Thanks for your amazing work in PartNet semantic segmentation. 

Could you please release the pre-trained models for the PartNet Semantic Segmentation (the fine-grained level)? 
It will contribute to the whole community and make it easier to follow your work.

**Comment by guochengqian** [comment]
Hello. @daerduoCarey 
Quick question.  For the PartNet Experiment. Did you use the best model on the validation dataset and report their performance on the test set?  Or you saved all the ckpts and used the best one among them on the test set? Or you used the ckpt from the last epoch / a certain epoch to test their performance on the test set.

**Comment by daerduoCarey** [comment]
I'm preparing some zip for you now. I simply use the last epochs once they've converged. There are too many ckpts to release as we are training one separate model per category/level.

**Comment by daerduoCarey** [comment]
Note that we are working on an online benchmark webpage for automatic evaluation and comparions to the existing numbers. Will be there by June.

**Comment by daerduoCarey** [comment]
Please check http://download.cs.stanford.edu/orion/partnet_dataset/pointcnn_sem_seg_pretrained_ckpts.zip (4.5G), updated on the webpage as well for the link.

We are now seriously working on a ScanNet-like online benchmark for PartNet/SAPIEN/LearingToGroup challenges. Thank you for your patiences.

**Comment by guochengqian** [comment]
> Please check http://download.cs.stanford.edu/orion/partnet_dataset/pointcnn_sem_seg_pretrained_ckpts.zip (4.5G), updated on the webpage as well for the link.
> 
> We are now seriously working on a ScanNet-like online benchmark for PartNet/SAPIEN/LearingToGroup challenges. Thank you for your patiences.

Wonderful. Thank you. Looking forward to the benchmark.
